### PR TITLE
I added a script to install kinetic-kane, as well as a generic worksp…

### PR DESCRIPTION
**Pull request by biocubed**
…ace generation script. All scripts are now executable
@lucbettaieb please look over the scripts and give any necessary feedback.

### CODE REVIEW

File: setup_ros_catkin_make_workspace.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+then
+  source ~/.bashrc
+  echo -e "\e[34m >>> Sourced ~/.bashrc to obtain the ros distro path. \e[39m"
+
+  mkdir -p ~/ros_ws/src
+	
+  cd ~/ros_ws/src  && catkin_init_workspace
+  cd ~/ros_ws && catkin_make
+
+  mkdir -p ~/ros_ws/src/cwru
```
Comment (lucbettaieb) [inline setup_ros_catkin_make_workspace.sh]: You're making these folders but not using them?

File: setup_ros_catkin_make_workspace.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  mkdir -p ~/ros_ws/src/local_code
+
+  git config --global user.name "$USERNAME"
+  git config --global user.email "$EMAIL"
+
+  cd ~/ros_ws/src/cwru && git clone https://github.com/cwru-robotics/cwru_msgs.git
```
Comment (lucbettaieb) [inline setup_ros_catkin_make_workspace.sh]: only cloning cwru_msgs?

Comment (lucbettaieb) [inline setup_ros_catkin_make_workspace.sh]: might wanna move this file into somewhere more specific and have it do more specific things

**Comment by lucbettaieb** [comment]
Take a look at the workspace setup script you added again.  I think you should make it set up a very specific workspace, or a very generic workspace (with nothing in it).

Kinetic script seems fine.

Thanks for chmodding

**Comment by biocubed** [comment]
I removed all cwru_specific commands.

**Pull request merged**